Source/RunActivity/Viewer3D/Viewer.cs

```diff
@@ -85,6 +85,8 @@ public class Viewer
         public HelpWindow HelpWindow { get; private set; } // F1 window
         public TrackMonitorWindow TrackMonitorWindow { get; private set; } // F4 window
         public HUDWindow HUDWindow { get; private set; } // F5 hud
+        public TrainDrivingWindow TrainDrivingWindow { get; private set; } // F5 train driving window
+        public MultiPlayerWindow MultiPlayerWindow { get; private set; } // MultiPlayer data windowed
```

Comment:
Is there a reason the multiplayer window has to be separate, but tied to the new train driving window? Why not bake its information into the existing train list window?

Reply:
The new train driving info window inherits the same principle as the HUD window. Like the original, the multiplayer information was displayed within the HUD window. 
Now the proposal is to show the multiplayer information in a dedicated window.
Maybe we could create a web-multiplayer window, if possible.

Reply:
Well, the HUD is a debugging tool not meant for use by ordinary players, so it makes sense to communicate the state of the multiplayer subsystem there.

But dedicating an entire window for the multiplayer state? One that can't be closed, shows just two lines, and contains information that seemingly would fit very well within the existing Train List (Alt+F9) window?

![image](https://user-images.githubusercontent.com/4701008/125154126-5150b400-e10d-11eb-9b4d-37677184e505.png)

I'm just having trouble seeing the justification here.

Reply:
> 
> 
> Well, the HUD is a debugging tool not meant for use by ordinary players, so it makes sense to communicate the state of the multiplayer subsystem there.
> 
> But dedicating an entire window for the multiplayer state? One that can't be closed, shows just two lines, and contains information that seemingly would fit very well within the existing Train List (Alt+F9) window?
> 
> ![image](https://user-images.githubusercontent.com/4701008/125154126-5150b400-e10d-11eb-9b4d-37677184e505.png)
> 
> I'm just having trouble seeing the justification here.

The Multiplayer info is used also by ordinary players, both by the dispatcher and by the clients (I have experienced that in a good amount of multiplayer sessions), so IMHO it makes sense to have that info together the Train Driving Info window. The info might however be included in the Train Driving Info window, instead of being in a separate, small window, although also the actual layout, with a separate add-on window, provides the same functionality from the point of view of the user.

Reply:
This is a multiplayer session, Dispatcher-player side:
![Multiplayerinfo-Dispatcher](https://user-images.githubusercontent.com/65355104/125192588-e7d5c000-e248-11eb-93e9-91f470e9c05e.jpg)
Similar info is replicated Client-player side.

Reply:
This would be the first window that is divided into multiple, interlinked windows, so the impetus is on you to explain why this would be desirable for players.

Reply:
Initially, the MP information was located after the Train Driving Info (TDI), elements, within the same window.
But the information of each new player who joined the MP session increased the width and height of the TDI window.
Taking into account, that the purpose of the abbreviated mode is to display a window as narrow as possible, keeping the information about the MP session in the same window, makes infeasible to obtain a narrow TDI window.
This was the reason to implement a dedicated Multiplayer window.